packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -828,6 +828,11 @@ export type CallWithChatEvent = 'callError' | 'chatError' | 'callEnded' | 'isMut
 // @public
 export type ChatAdapter = ChatAdapterThreadManagement & AdapterState<ChatAdapterState> & Disposable & ChatAdapterSubscribers & FileUploadAdapter;
 
+// @public
+export type ChatAdapterOptions = {
+    credential?: CommunicationTokenCredential;
```

Comment:
So we are providing the credential object instead, is it because it is easier to be refreshed? (I remember there is a optional refresh token embedded in this object)

Reply:
Yes. There is already a built in way to retrieve a new token if Contoso uses `RefreshTokenCredential`.